pkg/scheduler/factory/factory_test.go

```diff
@@ -236,6 +237,62 @@ func TestCreateFromEmptyConfig(t *testing.T) {
 	factory.CreateFromConfig(policy)
 }
 
+// Test configures a scheduler from a policy that does not contain any
+// predicates or priorities.
+// The predicates or priorities from DefaultProvider will be used.
+func TestCreateFromConfigWithEmptyPredicatesAndPriorities(t *testing.T) {
```

Comment:
We need an integration test to cover all the logic, including the decoding of policy config. Coding an integration test is similar. Please see [this](https://github.com/kubernetes/kubernetes/blob/master/test/integration/scheduler/scheduler_test.go#L83).

Reply:
Done.